src/themis/secure_message_wrapper.h

```diff
@@ -43,8 +43,6 @@ struct themis_secure_message_hdr_type {
 };
 typedef struct themis_secure_message_hdr_type themis_secure_message_hdr_t;
 
-#define THEMIS_SECURE_MESSAGE_LENGTH(hdr) ((hdr)->message_length)
```

Comment:
we don't use it anywhere else, do we?

Reply:
This macro call was removed from `src/themis/secure_message.c` in this same PR. And since everything compiles, I guess we don't need it anymore :+1:

Reply:
> we don't use it anywhere else, do we?

Yes, @iamnotacake is right on spot here.